bats_ai/settings.py

```diff
@@ -120,5 +122,18 @@ class HerokuProductionConfiguration(BatsAiMixin, HerokuProductionBaseConfigurati
     pass
 
 
-class AwsProductionConfiguration(BatsAiMixin, S3StorageMixin, _BaseConfiguration):
-    pass
+class AwsProductionConfiguration(BatsAiMixin, _BaseConfiguration):
+    DEFAULT_FILE_STORAGE = 'storages.backends.s3.S3Storage'
+    baseHost = '[batdetectai.kitware.com](http://batdetectai.kitware.com/)'
+    if 'SERVERHOSTNAME' in os.environ:
+        baseHost = os.environ['SERVERHOSTNAME']
+    ALLOWED_HOSTS = [baseHost]
+    CSRF_TRUSTED_ORIGINS = [f'https://{baseHost}', f'https://{baseHost}']
+    CORS_ORIGIN_WHITELIST = [f'https://{baseHost}', f'https://{baseHost}']
+
+    AWS_S3_REGION_NAME = 'us-west-2'
+    AWS_STORAGE_BUCKET_NAME = 'nabat-beta-batai-django'
```

Comment:
Lets move these into Environment variables because they might need to be changed.